errors/error.go

```diff
@@ -31,4 +31,6 @@ var (
 	ErrPeerLocalNode           = errors.New("The peer being added is the local node")
 	ErrProcessNotFound         = errors.New("The process is not running or is inaccessible")
 	ErrProcessAlreadyRunning   = errors.New("Process is already running")
+	ErrEtcdStartTimeout        = errors.New("Etcd embedded server took too long to start!")
```

Comment:
This is something I'd like to change. It's better to keep errors within the actual package returning these errors. It doesn't make sense to have a separate errors package (IMO).